### DISCUSSION TITLE
Bug: I can't run Litestar on Ubuntu

### DISCUSSION THREAD

**Discussion by juniozguedes** [litestar-org/litestar#2240] [Q&A] [answered]
### Description

I'm a new user. 
Created a virtual environment, a app.py file and installed litestar with pip install litestar

Copied the same code as the documentation provides as hello world

When I try to run the project with uvicorn app:app it returns to me:

    from litestar import Litestar, get
ModuleNotFoundError: No module named 'litestar'

### MCVE

```python
from litestar import Litestar, get
ModuleNotFoundError: No module named 'litestar'
```

### Steps to reproduce

```bash
1. Install litestar with pip install litestar
2. Run the project with uvicorn app:app
4. See error
```

### Screenshots

```bash
Same starter example as documentation
```

### Litestar Version

2.0.1fiinal0

### Platform

- [X] Linux
- [ ] Mac
- [ ] Windows
- [ ] Other (Please specify in the description above)

---

## Funding
* If you would like to see an issue prioritized, make a pledge towards it!
* We receive the pledge once the issue is completed & verified

<a href="https://polar.sh/litestar-org/litestar/issues/2239">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/litestar-org/litestar/issues/2239/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/litestar-org/litestar/issues/2239/pledge.svg">
</picture>
</a>

**Comment by provinzkraut** [comment]
This sounds like an issue with your virtual environment.
1. How did you create your venv?
2. Did you activate your venv?

**Comment by viett-appcard** [comment]
Hey, @provinzkraut , stumbling on this discussion, same issue but I don't see the optional dependency for [standard] as [mentioned in the tutorial](https://docs.litestar.dev/latest/tutorials/todo-app/0-application-basics.html#)

```shell
$ python3.11 -m venv venv
$ venv/bin/python -m pip install litestar[standard]
zsh: no matches found: litestar[standard]
```

```shell
❯ venv/bin/python -m pip install litestar[extra]
zsh: no matches found: litestar[extra]

❯ venv/bin/python -m pip install litestar[standard]
zsh: no matches found: litestar[standard]

❯ venv/bin/python -m pip install litestar[full]
zsh: no matches found: litestar[full]

❯ venv/bin/python -m pip install litestar[uvicorn]
zsh: no matches found: litestar[uvicorn]
```

**Reply by peterschutt** [reply, marked as answer]
Check this: https://stackoverflow.com/a/30539963/6560549
